spec/models/author_publication_spec.rb

```diff
@@ -5,4 +5,14 @@
   it 'has a publication' do
     expect(author_publication.publication).to be_a(Publication)
   end
+
+  subject do
+    AuthorPublication.new(email: 'test@test.edu', claim_link: 'test_claim_link') { |a| a.save!(validate: false) }
```

Comment:
The spec already creates an AuthorPublication using FactoryBot (see https://github.com/osulp/Easydeposit2/blob/master/spec/factories/author_publication.rb) .. You would introduce a new `claim_link` field to the factory, and then your test would check its value.. So you can eliminate the "subject do ... end" and have the expectations kinda like:
```
expect(author_publication[:email]).to eq 'test@test.edu'
```